### ISSUE TITLE
Move binary files out of version control

### ISSUE THREAD

**Issue by cup** [vtereshkov/xdpw#13]
Binary file should not exist here:

https://github.com/vtereshkov/xdpw/blob/1dff7ebc5d038951a8f971e7f1d378720b73ba35/xdpw.exe

Here instead:

https://github.com/vtereshkov/xdpw/releases

**Comment by vtereshkov** [comment]
Yes, I am considering it. But now it is convenient for me to clone just one folder and do whatever I want with it. Do you experience any problems with antivirus false positives when cloning a folder containing an .exe?
